backend/routes/cryptos.js

```diff
@@ -0,0 +1,204 @@
+var express = require('express');
+var router = express.Router();
+const client = require('../config/clientPg')
+
+function errorQuery(e, res) {
+    res.status(418).json({
+        error: `bdd request failed`,
+        message: `I'm a teapot`
+    })
+    console.error(e.stack)
+}
+
+/* GET cryptos infos
+/cryptos?[cmids={cm}]
+examples:
+    - /cryptos
+    - /cryptos?cmids=BTC
+    - /cryptos?cmids=BTC,ETH,XRP
+*/
+router.get('/cryptos', function (req, res, next) {
+    let query1 = "SELECT * from CRYPTO_LIST",
+        values = "";
+    if (req.query.cmids != undefined) {
+        const cmids = req.query.cmids.split(",");
+        values = `'${cmids[0]}'`
+        for (let i = 1; i < cmids.length; i++)
+            values += `,'${cmids[i]}'`
+        query1 = `SELECT id,symbol,fullname,picture_url from CRYPTO_LIST WHERE symbol IN (${values})`
+    }
+    client
+        .query(query1)
+        .then(result1 => {
+            let now = new Date()
+            let a = new Date(now.getFullYear(), now.getMonth(), now.getDate(), 1, 0, 0).getTime() / 1000
+            if (result1.rows.length == 0)
+                res.status(400).json({
+                    error: `No crypto matches symbols '${cmids}'`
+                })
+            else {
+                let query2 = `SELECT * from CRYPTO_HISTORY WHERE period = 'daily' AND crypto_id IN ( SELECT id FROM CRYPTO_LIST ) AND timestamp = to_timestamp(${a})`
```

Comment:
```suggestion
                const GET_CRYPTO_HISTORY_BY_DAY_AND_ID = `SELECT * from CRYPTO_HISTORY WHERE period = 'daily' AND crypto_id IN ( SELECT id FROM CRYPTO_LIST ) AND timestamp = to_timestamp(${a})`
```
Also, should be const, and to forget to change `to_timestamp(${a})` with the correct variable name